apps/ledger-live-desktop/src/renderer/screens/platform/App.jsx

```diff
@@ -20,17 +20,32 @@ type Props = {
     url: string,
   },
   appId?: string,
+  location: {
+    hash: string,
+    params: {
+      [key: string]: string,
+    },
+    pathname: string,
+    search: string,
+  },
 };
 
-export default function PlatformApp({ match, appId: propsAppId }: Props) {
+export default function PlatformApp({ match, appId: propsAppId, location }: Props) {
   const history = useHistory();
-  const { state: urlParams, search } = useLocation();
+  const { params: urlParams, search, pathname } = location;
   const appId = propsAppId || match.params?.appId;
+  const sufix = pathname.substr(
+    pathname.indexOf(match.params.appId) + match.params.appId.length + 1,
```

Comment:
Am I right in thinking we are trying to capture the query params from the `pathname` and then to add them to `manifest.params.dappUrl`? If so, is that data not available in the `search` variable?